frontend/ui/src/components/kanban-board/board.tsx

```
@@ -1,42 +1,344 @@
+import { useContext, useEffect, useRef, useState } from 'react';
+import invariant from 'tiny-invariant';
+import { Column } from './board-column';
+import {
+  isCardData,
+  isCardDropTargetData,
+  isColumnData,
+  isDraggingACard,
+  isDraggingAColumn,
+  TBoard,
+  TColumn,
+} from './types';
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)